tests/integration/helm_traces_load_balancing_test.go

```diff
@@ -1,101 +0,0 @@
-package integration
-
-import (
-	"context"
-	"fmt"
-	"testing"
-	"time"
-
-	appsv1 "k8s.io/api/apps/v1"
-	corev1 "k8s.io/api/core/v1"
-	"sigs.k8s.io/e2e-framework/klient/k8s/resources"
-	"sigs.k8s.io/e2e-framework/klient/wait"
-	"sigs.k8s.io/e2e-framework/klient/wait/conditions"
-	"sigs.k8s.io/e2e-framework/pkg/envconf"
-	"sigs.k8s.io/e2e-framework/pkg/features"
-
-	"github.com/stretchr/testify/require"
-
-	"github.com/SumoLogic/sumologic-kubernetes-collection/tests/integration/internal/ctxopts"
-	"github.com/SumoLogic/sumologic-kubernetes-collection/tests/integration/internal/stepfuncs"
-)
-
-func Test_Helm_Traces_Load_Balancing(t *testing.T) {
-	const (
-		tickDuration = time.Second
-		waitDuration = time.Minute * 2
-	)
-
-	featTraces := features.New("traces").
-		// TODO: Rewrite into similar step func as WaitUntilStatefulSetIsReady but for deployments
-		Assess("otelcol deployment is ready", func(ctx context.Context, t *testing.T, envConf *envconf.Config) context.Context {
-			res := envConf.Client().Resources(ctxopts.Namespace(ctx))
-			releaseName := ctxopts.HelmRelease(ctx)
-			labelSelector := fmt.Sprintf("app=%s-sumologic-otelcol", releaseName)
-			ds := appsv1.DeploymentList{}
-
-			require.NoError(t,
-				wait.For(
-					conditions.New(res).
-						ResourceListN(&ds, 1,
-							resources.WithLabelSelector(labelSelector),
-						),
-					wait.WithTimeout(waitDuration),
-					wait.WithInterval(tickDuration),
-				),
-			)
-			require.NoError(t,
-				wait.For(
-					conditions.New(res).
-						DeploymentConditionMatch(&ds.Items[0], appsv1.DeploymentAvailable, corev1.ConditionTrue),
-					wait.WithTimeout(waitDuration),
-					wait.WithInterval(tickDuration),
-				),
-			)
-			return ctx
-		}).
-		Assess("otelgateway deployment is ready", func(ctx context.Context, t *testing.T, envConf *envconf.Config) context.Context {
```

Comment:
It seems that this check is missing after the rewrite?

Reply:
Yeah, good eye catching that. I've added it back.